.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = {
+  root: true,
+  parser: '@typescript-eslint/parser',
+  plugins: ['@typescript-eslint'],
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:react/recommended',
+  ],
+  rules: {
+    'react/react-in-jsx-scope': 'off',
+    'semi': 'off',
+    '@typescript-eslint/semi': ['error', 'never'],
```

Comment:
I'm agnostic about this setting but I do like the simplicity disallowing the semi adds

Reply:
I don't want to push too hard, but I would prefer we kept the semicolons—there are situations where we'll _need_ it (which leaving it off will allow) and I'd rather we have it everywhere for consistency.

Reply:
e.g. writing this will require us to add a semicolon at the end of the first line:

```
foo = obj.foo;
({ foo } = obj)
```

Not that we will write this but as an example.

Reply:
Like I said I don't really have a preference. I added it back in.